source/mir/random/flex/internal/area.d

```diff
@@ -169,7 +172,8 @@ body
     {
         if (fabs(z) < constants!S.smallExp)
         {
-            area = exp(sh.a) * (iv.rx - iv.lx) * (1 + z / 2 + (z * z) / 6 + (z * z * z) / 24);
+            area = exp(sh.a) * (iv.rx - iv.lx) * (1 + z * S(0.5) + (z * z) * one_div_6
```

Comment:
Also, there's some duplication here: you can probably use `ivLength` instead of an explicit `iv.rx - iv.lx`, no ... ?